src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -259,19 +259,24 @@ impl Coordinator {
 
                     self.maybe_create_linked_cluster(source_id).await;
 
+                    let register_ts = self.get_local_write_ts().await.timestamp;
                     self.controller
                         .storage
-                        .create_collections(vec![(
-                            source_id,
-                            CollectionDescription {
-                                desc: source.desc.clone(),
-                                data_source,
-                                since: None,
-                                status_collection_id,
-                            },
-                        )])
+                        .create_collections(
+                            register_ts,
+                            vec![(
+                                source_id,
+                                CollectionDescription {
+                                    desc: source.desc.clone(),
+                                    data_source,
+                                    since: None,
+                                    status_collection_id,
+                                },
+                            )],
+                        )
                         .await
                         .unwrap_or_terminate("cannot fail to create collections");
+                    self.apply_local_write(register_ts).await;
```

Comment:
For all the calls to `apply_local_write()`, all other tables will not be readable at the current oracle timestamp until the next group commit. So we should either advance the upper of all other tables or trigger a group commit.

Reply:
Done!